media-tv/mythtv/files/mythtv-35.Fix-Qt6-compilation-on-Fedora-rawhide.patch

```diff
@@ -0,0 +1,108 @@
+From 80ebec6e2760f5d2a5faccdf3904cb35514cb318 Mon Sep 17 00:00:00 2001
```

Comment:
Missing a link to the commit and any relevant Gentoo bugs at the top. Please _always_ include these. Can you add them in a followup? Thanks.

It is important for a few reasons but especially to help other distros.

Reply:
yes